.storybook/components/ColorList/ColorList.module.css

```diff
@@ -3,7 +3,6 @@
   padding: calc(var(--eds-spacing-size-half) / 16 * 1rem);
   border-radius: calc(var(--eds-theme-box-button-border-radius) * 1px);
 
-  font: var(--eds-theme-typography-label-md-subtle);
```

Comment:
removing the tokenized imports throughout. instead of having these CSS-only values in the token sheet, we embed the combinations of primitive tokens in the `Heading` and `Text` components, and a few places inline when more convenient.

Will document the rationale and exceptions more deeply in the Contribution MDX file